README.md

```diff
@@ -1,3 +1,72 @@
+# How to Build and Test a Node
+
+To build and test your Waves Node, you will need to follow these steps \(Installation is possible only on _**Ubuntu**_, because, sbt packageAll ‌produces only deb package\).
+
+## 1. Setup the environment
```

Comment:
I suggest replacing Linux-specific environment setup instructions with a link to SBT installation instructions ([Mac](https://www.scala-sbt.org/1.0/docs/Installing-sbt-on-Mac.html), [Windows](https://www.scala-sbt.org/1.0/docs/Installing-sbt-on-Windows.html), [Linux](https://www.scala-sbt.org/1.0/docs/Installing-sbt-on-Linux.html))